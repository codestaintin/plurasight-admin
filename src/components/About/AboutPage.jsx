import React from 'react';
/**
 *
 * @function AboutPage
 *
 * @returns {JSX}
 */
const AboutPage = () => (
  <div className="app jumbotron">
    <h1>About</h1>
    <p>
      This application uses React, Redux, React Router and a variety of other
      helpful libraries
    </p>
  </div>
);

export default AboutPage;
