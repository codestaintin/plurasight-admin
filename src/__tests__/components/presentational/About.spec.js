import React from 'react';
import Adapter from 'enzyme-adapter-react-16';
import { configure, shallow } from 'enzyme';
import AboutPage from '../../../components/About/AboutPage';

configure({ adapter: new Adapter() });

describe('<AboutPage>', () => {
  it('renders the <AboutPage> component', () => {
    const tree = shallow(<AboutPage />);
    expect(tree).toMatchSnapshot();
  });
});
