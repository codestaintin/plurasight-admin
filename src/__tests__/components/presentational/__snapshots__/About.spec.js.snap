// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AboutPage> renders the <AboutPage> component 1`] = `
<div
  className="app jumbotron"
>
  <h1>
    About
  </h1>
  <p>
    This application uses React, Redux, React Router and a variety of other helpful libraries
  </p>
</div>
`;
